_data/chains/eip155-500.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Camino C-Chain",
+  "chain": "CAM",
+  "rpc": ["https://mainnet.camino.foundation/ext/bc/C/rpc"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Camino",
+    "symbol": "CAM",
+    "decimals": 18
+  },
+  "infoURL": "https://camino.foundation/",
+  "shortName": "Camino",
+  "chainId": 500,
+  "networkId": 1000,
+  "icon": "camino",
+  "explorers": [
+    {
+      "name": "blockexplorer",
+      "url": "https://explorer.camino.foundation",
```

Comment:
can you provide block links for this explorer?

Reply:
What do you mean by "block links"? Can you please elaborate?

Reply:
please read EIP3091